src/views/Auth/Auth.jsx

```diff
@@ -0,0 +1,74 @@
+import React, { useState } from 'react';
+import { useHistory } from 'react-router-dom';
+import { useAuth } from '../../hooks/useAuth';
+import { signInUser, signUpUser } from '../../services/user';
+
+import styles from './Auth.css';
+
+export default function Auth() {
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const [error, setError] = useState('');
+  const [signInUp, setSignInUp] = useState('Sign-In');
+
```

Comment:
Let's have this state live within the useAuth hook to uphold the architecture requirements.